taglib/ape/apeproperties.cpp

```diff
@@ -137,7 +137,7 @@ void APE::Properties::read(File *file, offset_t streamLength)
     analyzeOld(file);
 
   if(d->sampleFrames > 0 && d->sampleRate > 0) {
-    const double length = d->sampleFrames * 1000.0 / d->sampleRate;
+    const auto length = static_cast<double>(d->sampleFrames) * 1000.0 / d->sampleRate;
```

Comment:
What does the `static_cast` add here? Since `1000.0` is a double `d->sampleFrames` is promoted to a double before multiplication.

Reply:
It’s an implicit conversion that -Wconversion warns on. -Wconversion is added by utf8cpp because of…reasons

Reply:
Fair enough. I suppose I don't consider a promotion to double in the same way as an implicit narrowing conversion but it doesn't hurt to make it explicit.